README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 01
+# Pete Cuppernull 
+
+## Computing for the Social Sciences
+
+## Homework Assignment 1
+
+### **Biography**
+
+My name is **Pete Cuppernull** and I am a first year PhD student in the political science department. I'm studying international relations and quantitative methods and plan to complete the CSS certificate program. I'm interested in studying humanitarian interventions and specifically how states choose to pursue different types of interventions abroad. 
+
+Before coming to UChicago, I lived in New York City. When I first moved to New York, I worked in the financial technology industry where I researched corporate governance and shareholder activism. I ultimately transitioned to a product development role at my firm where I designed a few data collection applications and gained experience with SQL and Python. I left the finance world in 2017 to pursue my other passion in music, and worked as a music producer in the city for a couple of years. I've built an artist project with my older brother producing deep house and techno music, and I have worked as a sound engineer for pop artists as well. A handful of artists I've worked for are: 
+
+* The Chainsmokers
+* Diplo 
+* Billie Eilish
+
+After exploring these other interests, I decided to return to school to pursue my education in international relations and peace studies. I'm excited to continue to build my computational skills in this class and look forward to a great quarter!
+
+Here is a photo of myself. 
+
+![Here is a photo of myself.](photo.jpg)
+
+
+[Here is a link to the UChicago Homepage](https://www.uchicago.edu/)
```

Comment:
You can do it like `[Here] is a link to the UChicago Homepage(https://www.uchicago.edu/)`. This link seems a bit out of place.